Readme.md

```diff
@@ -1,57 +1,64 @@
 3. Tutorial - technically explain how to complete a certain task.
 4. Case study - present how you (or the company you work for) have used this tool, or are planning to use it.
 
+To get these external contributions recognized for our Hacktoberfest swag prizes, you'll need to create an Issue and then a Pull Request here in this Hacktoberfest repo - read on for more details. 
+
 ## Contribution workflow
 
-1. Only contributions to the participating repos will be considered.
-2. Eligible contributions must start with a Github Issue describing the work item.
-    1. For any contribution, first look for an existing Issue, and if needed open an Issue in the project's repo.
-    2. Issue can also be opened in this repository ([Hacktoberfest](https://github.com/aquasecurity/Hacktoberfest)) if they don't belong in the project's repo.
-    3. For non-code contributions, please create the issue in this repository ([Hacktoberfest](https://github.com/aquasecurity/Hacktoberfest)).
+1. Only contributions related to the participating projects listed above will be considered.
+2. Eligible contributions **must start with a Github Issue describing the work item**.
+    1. For any contribution, first look for an existing Issue. If it doesn't already exist, open a new Issue in the project's repo. 
+    2. For external contributions, please create an issue in this repository ([Hacktoberfest](https://github.com/aquasecurity/Hacktoberfest)) describing what you're planning to do. 
 3. Create a Pull Request with your contribution.
     1. Read the contributions guidelines for the relevant project.
     2. Link to the relevant Issue in the PR description.
     3. External contributions: please create the PR to the [contrib-integrations.md](contrib-integrations.md) document adding your work.
     4. Non-code contributions: please create the PR to the [contrib-content.md](contrib-content.md) document adding your work.
-4. A maintainer will label eligible PRs with the `#hacktoberfest-approved` label.
-5. Claim your prize by filling out the [following form](https://forms.office.com/Pages/ResponsePage.aspx?id=80wDvGtWykGfJF3ElHSwXoMzxQ44cLZDuLrHx6o4yX1UNklOSjVNOVFCSUtMVkVKR1VEU1haQVpUOS4u).
+4. A maintainer will label eligible PRs with the `#hacktoberfest-approved` label. 
+    1. PRs that don't correspond to an existing Issue are not eligible. 
+    2. PRs that don't relate to the participating projects are unlikely to be approved, but please feel free to raise ideas here in the [Discussions](https://github.com/aquasecurity/Hacktoberfest/discussions) tab. 
+    2. PRs that add meaningful improvements to docs are great! But swag-hunters please note: PRs that make only superficial changes will not be eligible. For example: simply adding or removing punctuation is not eligible, but adding a missing heading into a table of contents or fixing a broken link is eligible. In all cases, there needs to be a corresponding issue for your PR. 
```

Comment:
maybe this belongs in the "We are looking for meaningful contribution..." section above?

Reply:
I'm going to merge this and we can address this comment later if you think so